modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc
+:_content-type: PROCEDURE
+
+[id="op-using-custom-scc-to-improve-prod-security_{context}"]
+= Using a custom SCC to improve pod security
+
+To improve the security of your builds and runs, you can use custom security context constraints (SCCs).
+
+.Procedure
+
+Create a `pipelines-scc` Security Context Constraints (SCC) and configure its `runAsUser` and `seLinuxContext` fields.
++
+.Example `pipelines-scc` object
```

Comment:
I would consider removing this heading entirely and merging this text into the procedural step. Maybe change the step text to something like: `Using the following example as a model, reate a `pipelines-scc` Security Context Constraints (SCC) and configure its `runAsUser` and `seLinuxContext` fields:`